src/matrices/skeletonization.jl

```diff
@@ -0,0 +1,396 @@
+#=
+In this file the core computations for skeletonizing a recurrence matrix
+are defined.
+=#
+
+# Core function, which gets exported
+"""
+    skeletonize(R) → R_skel
+
+Skeletonizes the recurrence matrix `R` (see [`recurrence_matrix`](@ref)) by using
+the algorithm proposed by Kraemer & Marwan [^Kraemer2019]. This function returns
+`R_skel`, a recurrence matrix, which only consists of diagonal lines of
+"thickness" one.
+
+## References
```

Comment:
You don't need the header `## References` because the references go to the end of the documentation once compiled.